pantheon-sessions.php

```diff
@@ -261,6 +261,206 @@ public static function force_first_load() {
 			}
 		}
 	}
+
+	/**
+	 * Checks whether primary keys were set and notifies users if not.
+	 */
+	public static function check_native_primary_keys() {
+		global $wpdb;
+		$table_name = $wpdb->base_prefix . 'pantheon_sessions';
+		$old_table  = $wpdb->base_prefix . 'bak_pantheon_sessions';
+		$query      = "SHOW KEYS FROM {$table_name} WHERE key_name = 'PRIMARY';";
+
+		$key_existence = $wpdb->get_results( $query );
+
+		if ( empty( $key_existence ) ) {
+			// If the key doesn't exist, recommend remediation.
+			?>
+			<div class="notice notice-error is-dismissible">
+				<p>
+				<?php
+				print __( 'Your PHP Native Sessions table is missing a primary key. Please run "wp {site_name}.dev pantheon session add-index" and verify that the process completes successfully and that this message goes away, then run "wp {site_name}.live pantheon session add-index" to resolve this issue on your live environment.',
+				'wp-native-php-sessions' );
```

Comment:
arguably worth using `<code>` tags here for the WP CLI functions rather than `"`.
Also: this plugin is used outside of Pantheon environments. So we should not assume that `wp site_name.dev` is valid, since `site_name.dev` is a Pantheon-ism. 

```suggestion
				print wp_kses_post( __( 'Your PHP Native Sessions table is missing a primary key. Please run <code>wp pantheon session add-index</code> and verify that the process completes successfully and that this message goes away to resolve this issue on your live environment.', 'wp-native-php-sessions' ) );
```

Alternately, we could potentially add code that checks for a Pantheon environment and displays your message if we're on a Pantheon environment and the above message otherwise.